slash_commands/auth.py

```diff
@@ -1,8 +1,132 @@
 import discord
 import os
+import secrets
 from APIs.SweccAPI import SweccAPI
 
 swecc = SweccAPI()
+VERIFIED_ROLE_ID = int(os.getenv("VERIFIED_ROLE_ID"))
+
+class RegisterModal(discord.ui.Modal, title="Register Your Account"):
+    def __init__(self, bot_context, verified_role):
+        super().__init__(timeout=None)
+        self.bot_context = bot_context
+        self.verified_role = verified_role
+        self.OFF_TOPIC_CHANNEL_ID = int(os.getenv("OFF_TOPIC_CHANNEL_ID"))
```

Comment:
Can we use context here since it's importing all the env variables. I kind of like that style instead importing the env variables in every file.